smartmeter/display.cpp

```diff
@@ -22,36 +22,22 @@ std::vector<String> Display::strings = {
 };
 
 Display::Display() :
+    
     // Initialize the OLED display using brzo_i2c
     // D3 -> SDA
     // D5 -> SCL
-    display(new SSD1306Brzo(0x3c, D3, D5))
+    display(new SSD1306Brzo(0x3c, SDA, SCL))
```

Comment:
This is different from the prototype.

Reply:
It should not have passed on the PR. Corrected.